plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1078,4 +1078,232 @@ SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]\.php/apps/files/api/v[0-9]/transferown
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
 
+#
+# [ Nextcloud Mail ]
+#
+
+# Some email addresses end in .com or .inc
+SecRule REQUEST_FILENAME "@rx /apps/mail/api/autoconfig/ispdb/[^/]+(?:com|inc)$" \
+    "id:9508970,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ctl:ruleRemoveById=920440,\
```

Comment:
This and also rule below (`9508971`) should be fixed by similar way as `allowed_methods`, see rules 9508100 and for example `9508122`.

Reply:
I'm not sure how that would work, I'm not adding a restricted file extensions like in ``9508122`` but removing? it should respect any changes made by the end user since it only matches file extensions ending in ``(com|inc)``

Reply:
I just realised it probably cannot be done as we need to remove something from a list, not add it. Any ideas?

Reply:
Ok, let's keep it like it is now.